### PR TITLE
Implement distribution viewer by date using API.

### PR DESCRIPTION
**Pull request by rlr**
r? @ncloudioj 

To test:

```
$ fab build_ui
$ manage.py runserver
$ open http://127.0.0.1:5000/distribution/
```

### CODE REVIEW

File: ui/src/actions/WebtilesPreviewer.js [hunk 1]
```diff
@@ -29,97 +36,68 @@ export function selectType(tileType) {
...
 
-function shouldFetchTiles(state, channel, locale) {
-  const localeObj = state.channels[channel].locales[locale];
-  if (localeObj && !localeObj.directoryTiles && !localeObj.isFetching) {
-    return true;
-  }
-  return false;
-}
+            // Get the artifact key
+            let agUrl = index[locale].ag;
```
Comment (ncloudioj) [inline ui/src/actions/WebtilesPreviewer.js]: The new distribution API will return a list of URLs both for the `ag` and `legacy`. Is this only for the current tile index now?

Comment (rlr) [inline ui/src/actions/WebtilesPreviewer.js]: Yeah. When you land an updated API, I'll fix that shortly after. Sounds good?

Comment (ncloudioj) [inline ui/src/actions/WebtilesPreviewer.js]: Sounds good!

I've already merged that PR to sp2 branch. Watch out! :smirk_cat:

Comment (rlr) [inline ui/src/actions/WebtilesPreviewer.js]: oh... Well let me just add a check here and for now will only use the first item in array. then will fix for real it in a new PR.

**Comment by rlr** [comment]
fixed that ^

File: ui/src/actions/WebtilesPreviewer.js (hunk 1 above)
Comment (ncloudioj) [inline ui/src/actions/WebtilesPreviewer.js]: :laughing: oops, short notice.

My bad. Amigo.

**Comment by ncloudioj** [comment]
Once coveralls gets the job done r+!

**Pull request closed without merging**